master/pkg/tasks/task.go

```diff
@@ -224,6 +224,14 @@ func (t TaskSpec) EnvVars() map[string]string {
 	return e
 }
 
+// LogShipperWrappedEntrypoint returns the configured Entrypoint wrapped with ship_logs.py.
+func (t *TaskSpec) LogShipperWrappedEntrypoint() []string {
+	// Prepend the entrypoint like: `ship-logs.sh ship_logs.py "$@"`.
+	shipLogsShell := filepath.Join(RunDir, taskShipLogsShell)
+	shipLogsPython := filepath.Join(RunDir, taskShipLogsPython)
+	return append([]string{shipLogsShell, shipLogsPython}, t.Entrypoint...)
+}
+
 // ToDockerSpec converts a task spec to a docker container spec.
 func (t *TaskSpec) ToDockerSpec() cproto.Spec {
```

Comment:
it is kinda funny to me that ```ToDockerSpec``` is here and not under ```internal/rm/agentrm```

created https://hpe-aiatscale.atlassian.net/browse/DET-9894 for backend purposes

Reply:
Thanks, I also thought that was weird.  That's how I missed this in the first place.